sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -63,6 +63,8 @@ struct sof_sdw_codec_info {
 		     struct sof_sdw_codec_info *info,
 		     bool playback);
 
+	int (*exit)(struct device *dev);
+	bool is_probed;
```

Comment:
is_probed is confusing. The codec is probed based on ACPI stuff, not matter what happens in the machine driver.

This should be 'initialized' maybe.